lsif/src/shared/store/uploads.ts

```diff
@@ -54,11 +58,30 @@ export class UploadManager {
         visibleAtTip: boolean,
         limit: number,
         offset: number
-    ): Promise<{ uploads: pgModels.LsifUpload[]; totalCount: number }> {
-        const [uploads, totalCount] = await instrumentQuery(() => {
+    ): Promise<{ uploads: LsifUploadWithPlaceInQueue[]; totalCount: number }> {
+        const {
+            uploads,
+            raw,
+            totalCount,
+        }: {
+            uploads: pgModels.LsifUpload[]
+            raw: { upload_id: number; rank: string | undefined }[]
+            totalCount: number
+        } = await instrumentQuery(async () => {
```

Comment:
Shouldn't the type here be inferred? If for some reason it can't be, I'd still prefer it as a type parameter:

```suggestion
        } = await instrumentQuery<{
            uploads: pgModels.LsifUpload[]
            raw: { upload_id: number; rank: string | undefined }[]
            totalCount: number
        }>(async () => {
```

Reply:
raw's type is any[], but we know better based on the query shape.